ORStools/gui/ORStoolsDialog.py

```diff
@@ -47,7 +48,15 @@
 
 from PyQt5.QtCore import QSizeF, QPointF, QCoreApplication, QSettings
 from PyQt5.QtGui import QIcon, QTextDocument
-from PyQt5.QtWidgets import QAction, QDialog, QApplication, QMenu, QMessageBox, QDialogButtonBox
+from PyQt5.QtWidgets import (
```

Comment:
```suggestion
from qgis.PyQt.QtWidgets import (
```
This is best practice for qgis imports of PyQt. It'll help with the Qt6 migration, and also there's some wrappers in the qgis import which fix some things.